radis/phys/units.py

```diff
@@ -114,34 +106,29 @@ def conv2(quantity, fromunit, tounit):
     """
 
     try:
-        a = Q_(quantity, fromunit)
-        a = a.to(tounit)
+        a = quantity * u.Unit(fromunit)
+        a = a.to(u.Unit(tounit))
 
         # Hardcoded: 'pint' considers angles to be dimensionless (which they are)
```

Comment:
I think unlike pint, astropy handles Angles as a physical unit by default. So the comment could be removed. 

Add to the test something like 
```
assert mW/cm2/sr/nm not equivalent to mW/cm2/nm  ?  # pseudo-code
```